distro/pom.xml

```
@@ -34,7 +34,7 @@
     <openconceptlab.version>2.4.0</openconceptlab.version>
     <attachments.version>3.7.0</attachments.version>
     <referencedemodata.version>2.5.0</referencedemodata.version>
-    <queue.version>2.7.0-SNAPSHOT</queue.version>
+    <queue.version>2.8.0-SNAPSHOT</queue.version>
```

```suggestion
    <queue.version>2.7.0</queue.version>
```

**The version here should be set to 2.7.0.**

_But WHY???_

We generally use snapshot versions only when we need to pull in unreleased changes from dev3. Snapshots are useful for testing features or bug fixes that haven’t yet made it into an official release. However, they can introduce instability because the snapshot may change over time, and it’s not guaranteed to be production-ready.

In this case, we don’t need any unreleased changes. That means we should stick with the latest stable released version, which is 2.7.0. Using the released version ensures consistency, stability, and reproducibility across environments.

Just wanted to highlight this so it’s clear why we’re making this choice, in future PRs, if you’re unsure whether to use a snapshot or a released version, a good rule of thumb is:

_Released version_ → whenever possible (default choice).

_Snapshot version_ → only when you explicitly need unreleased changes from dev3.